integration/flink/flink115/build.gradle

```diff
@@ -0,0 +1,139 @@
+/*
+/* Copyright 2018-2023 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+plugins {
+    id 'java'
```

Comment:
See my previous note about the necessity of `java` when you're applying `java-library`.